<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabRegionalPeopleCountingSettings</class>
 <widget class="QWidget" name="TabRegionalPeopleCountingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="WidgetMessage" name="widgetMessage" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButtonCopy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="ButtonGroup" name="widgetButtonGroup" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widgetSize" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonMaxLayout">
        <property name="text">
         <string>Maximum Size(1x1~320x240)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="VCASizeLineEditBox" name="widgetMaxLineBox" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonMinLayout">
        <property name="text">
         <string>Minimum Size(1x1~320x240)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="VCASizeLineEditBox" name="widgetMinLineBox" native="true"/>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="CommonVideo" name="commonVideo" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>645</width>
        <height>328</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetSettings" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSetRegion">
            <property name="text">
             <string>Region</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelRegionalEnable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Regional People Counting</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelSensitivity">
            <property name="text">
             <string>Sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="CheckableLineEdit" name="checkableLineEditMaxLength" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="BlueLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>1~1800s</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="widgetRegion" native="true">
            <layout class="QHBoxLayout" name="horizontalLayoutRegion">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PushButtonEditState" name="pushButtonRegionEdit">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MyPushButton" name="pushButtonRegionSetAll">
               <property name="text">
                <string>Set All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MyPushButton" name="pushButtonRegionDelete">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QWidget" name="widgetRegionalPeopleCountingSchedule" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="MyPushButton" name="pushButtonRegionalPeopleCountingSchedule">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>310</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <widget class="QWidget" name="widgetMaxStay" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="CheckableLineEdit" name="checkableLineEditMaxStay" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="BlueLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>1~60</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="MyCheckBox" name="checkBoxEnable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelObjectSizeLimits">
            <property name="text">
             <string>Object Size Limits</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="CheckableLineEdit" name="checkableLineEditMinStay" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="BlueLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>1~60</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxRegion"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelMinStay">
            <property name="text">
             <string>MIN.Stay</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QWidget" name="widgetObjectSize" native="true">
            <layout class="QHBoxLayout" name="horizontalLayoutObjectSize">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PushButtonEditState" name="pushButtonObjectSizeEdit">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MyPushButton" name="pushButtonObjectSizeReset">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelRegionalPeopleCountingSchedule">
            <property name="text">
             <string>Regional People Counting Schedule</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="BaseSlider" name="sliderSensitivity">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelMaxLength">
            <property name="text">
             <string>MAX.length of Stay</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QWidget" name="widgetAction" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="MyPushButton" name="pushButtonEditAction">
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="labelAction">
            <property name="text">
             <string>Action</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelMaxStay">
            <property name="text">
             <string>MAX.Stay</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelRegionNumber">
            <property name="text">
             <string>Region No.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BaseSlider</class>
   <extends>QSlider</extends>
   <header>baseslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMessage</class>
   <extends>QWidget</extends>
   <header>widgetmessage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CommonVideo</class>
   <extends>QWidget</extends>
   <header>commonvideo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header>buttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlueLabel</class>
   <extends>QLabel</extends>
   <header>bluelabel.h</header>
  </customwidget>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header>MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>PushButtonEditState</class>
   <extends>QPushButton</extends>
   <header>pushbuttoneditstate.h</header>
  </customwidget>
  <customwidget>
   <class>VCASizeLineEditBox</class>
   <extends>QWidget</extends>
   <header>VCASizeLineEditBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckableLineEdit</class>
   <extends>QWidget</extends>
   <header>CheckableLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
